main.py

```diff
@@ -1113,11 +1113,13 @@ def __call__(
         environ: Dict[str, str],
         start_response: webapp2.Response
     ) -> webapp2.Response:
+        # Here we use MyPy ignore because cache_services is of ModuleType
+        # and ModuleType does not contain 'CLOUD_NDB_REDIS_CLIENT' attribute.
         global_cache = datastore_services.RedisCache(
             cache_services.CLOUD_NDB_REDIS_CLIENT)  # type: ignore[attr-defined]
         with datastore_services.get_ndb_context(global_cache=global_cache):
-            # Cast is needed since webapp2.WSGIApplication is not
-            # correctly typed.
+            # Here we use cast because webapp2.WSGIApplication is not
+            # correctly typed in it's stubs.
```

Comment:
... in its stubs.

But again, can we fix the stubs?

Reply:
There is no stubs for `webapp2.WSGIApplication`. I tried but i didn't find any

Reply:
Fixed

Reply:
@sahiljoster32 Did you see this comment?

Reply:
Yeah, I already did changes!